azure/function-app/communication-service/README.md

```diff
@@ -1,5 +1,46 @@
 # communication-service
 
-This example provisions a Function App that can connect to an Azure Communication Service bundle. This example is using the Azure Email Communication Services resource, which is still in preview. Provision an Email service resource in the Azure portal [here](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Communication%2FEmailServices).
+This example provisions a Function App that can connect to an Azure Communication Service bundle. This example is using the Azure Email Communication Services resource, which is still in public preview. 
```

Comment:
Nit, don't worry! What do you think about adding a link to the doc page?
https://learn.microsoft.com/en-us/azure/communication-services/concepts/email/email-overview

Idea being that it will probably leave preview before we update these. The link will be "always live".